content/instructor/collections/index.mdx

```diff
@@ -9,6 +9,8 @@ shareImage: 'https://res.cloudinary.com/dg3gyk0gu/image/upload/v1571260698/og-im
 
 If you want a quick way to group together a set of related lessons you can create a collection. A collection is a playlist of videos that can share a common theme and be shared and promoted to your audience.
 
+![web browser displaying egghead collection page which includes a title, a description, and a list of videos](/images/collections-screenshot.png)
```

Comment:
@hiroko this needs to be a relative link. Otherwise, it will look all the way from the root. That is done with `./` instead of simply `/`

```suggestion
![web browser displaying egghead collection page which includes a title, a description, and a list of videos](./images/collections-screenshot.png)
```